ocaml/networkd/bin_db/networkd_db.ml

```diff
@@ -31,22 +35,62 @@ let _ =
     (Printf.sprintf "Usage: %s [-bridge <bridge> | -iface <interface>]" name) ;
   try
     let config = Network_config.read_config () in
-    if !bridge <> "" then
-      if List.mem_assoc !bridge config.bridge_config then (
-        let bridge_config = List.assoc !bridge config.bridge_config in
-        let ifaces =
-          List.concat_map (fun (_, port) -> port.interfaces) bridge_config.ports
-        in
-        Printf.printf "interfaces=%s\n" (String.concat "," ifaces) ;
-        match bridge_config.vlan with
+    let r =
+      let* bridge = if !bridge = "" then Error Skip else Ok !bridge in
+      let* bridge_config =
+        match List.assoc_opt bridge config.bridge_config with
```

Comment:
This could use ` |> Result.fold ~none:error ~some:Result.ok` instead of the match